doc/contribute/index.rst

```diff
@@ -128,19 +128,35 @@ below and at http://developercertificate.org/.
         consistent with this project or the open source license(s)
         involved.
 
-DCO Sign-Off Methods
-====================
+DCO Sign-Off
+============
 
-The DCO requires a sign-off message in the following format appear on each
-commit in the pull request::
+The "sign-off" in the DCO is a "Signed-off-by:" line in each commit's log
+message. The Signed-off-by: line must be in the following format::
 
    Signed-off-by: Zephyrus Zephyr <zephyrus@zephyrproject.org>
 
-The DCO text can either be manually added to your commit body, or you can add
-either ``-s`` or ``--signoff`` to your usual Git commit commands. If you forget
-to add the sign-off you can also amend a previous commit with the sign-off by
-running ``git commit --amend -s``. If you've pushed your changes to GitHub
-already you'll need to force push your branch after this with ``git push -f``.
+For your commits, replace:
+
+- ``Zephyrus Zephyr`` with your legal name (pseudonyms, hacker handles, and the
+  names of groups are not allowed)
+
+- ``zephyrus@zephyrproject.org`` with the same email address you are using to
```

Comment:
ditto, some common email place holders would be preferable or skip it, everyone knows how an email looks like I guess. Someone might think they need to use this one.